src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -799,11 +799,11 @@ internal static void DecryptKey(SafeKeyHandle safeKeyHandle, byte[] encryptedDat
             VerifyValidHandle(safeKeyHandle);
             if (null == encryptedData)
             {
-                throw new CryptographicException(SR.Format(SR.Argument_InvalidValue, "Encrypted Data is null"));
+                throw new CryptographicException(SR.Argument_InvalidValue + " Encrypted Data is null");
             }
             if (encryptedDataLength < 0)
             {
-                throw new CryptographicException(SR.Format(SR.Argument_InvalidValue, "Encrypted data length is less than 0"));
+                throw new CryptographicException(SR.Argument_InvalidValue + " Encrypted data length is less than 0");
```

Comment:
These changes aren't localizable.  Which sent me on a hunt.

In netfx these two things were checked inside the CLR via a PRECONDITION tests in a QCall.  That turns out to be equivalent to Debug.Assert().  So the two if/throws here should just be changed to Debug.Assert(the opposite of what the if says).